spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'order'
+
+describe Order do
+  subject(:order)     { described_class.new("2 Chicken Ruby", "1 Kingfisher") }
+  let(:order_hash)    { { chicken_ruby: 2, kingfisher: 1 } }
+
+  describe '#initialize' do
+    context 'when ordering an incorrect dish' do
+      it 'raises an error' do
+        expect { described_class.new("7 Incorrect Dish") }
+          .to raise_error (/Sorry, we do not have/)
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)<br>Style/RedundantParentheses: Don't use parentheses around a literal.